code/modules/mob/new_player/new_player_login.dm

```diff
@@ -29,8 +29,9 @@
 	if((ckey in GLOB.de_admins) || (ckey in GLOB.de_mentors))
 		client.verbs += /client/proc/readmin
 
-	client?.playtitlemusic()
-	client?.update_active_keybindings()
+	client.display_job_bans(TRUE)
```

Comment:
This will probably hit the database a fair bit, right? Could be a lot of queries on round restart.

Reply:
Tweaked behavior to not reload job bans when automatically called as part of joining, since we already load the data from DB as part of new client stuff